### PR TITLE
Use new Go module name for generated files

### PR DESCRIPTION
**Pull request by timwangmusic**
These are generated with `bash hack/make-rules/update/codegen.sh` command.

/assign: @airbornepony

### CODE REVIEW

File: prow/plugins/plugin-config-documented.yaml
Comment (airbornepony) [inline prow/plugins/plugin-config-documented.yaml:1]: This is weird. I recall this change--it should be a move of this block to a different location in the generated file, not a strict delete.

Was the block deleted after https://github.com/kubernetes/test-infra/pull/31619/files was merged?

Comment (airbornepony) [reply prow/plugins/plugin-config-documented.yaml:1]: I just confirmed this block is still present. Not sure why it's being deleted.

File: prow/config/prow-config-documented.yaml
Comment (airbornepony) [inline prow/config/prow-config-documented.yaml:1]: These deletions don't seem right.

**Comment by timwangmusic** [comment]
See #82

**Pull request closed without merging**